pywebdriver/views.py

```diff
@@ -105,3 +105,12 @@ def system():
 @app.route("/static/images/<path:path>", methods=["POST", "GET", "PUT", "OPTIONS"])
 def image_html(path=None):
     return app.send_static_file(os.path.join("images/", path))
+
+
+@app.route("/shutdown", methods=["GET"])
+def shutdown(path=None):
+    shutdown_func = request.environ.get("werkzeug.server.shutdown")
+    if shutdown_func is None:
+        raise RuntimeError("Not running werkzeug")
+    shutdown_func()
+    return "Shutting down..."
```

Comment:
@PierrickBrun Sorry to be late to the party... but I think this has security implications (remote shutdown) ? 
Also [a GET should never have side effects](https://twitter.com/rombulow/status/990684453734203392?lang=en). ;)

Reply:
@sbidoul  I'll try to see if I can do it with a better approach (like a thread/process) or I'll fallback to this with a POST